cmd/module/cmd.go

```diff
@@ -18,10 +18,12 @@ func main() {
 
 func mainWithArgs(ctx context.Context, _ []string, logger logging.Logger) error {
 	if logger.GetLevel() == logging.DEBUG {
-		videostore.SetLibAVLogLevel("info")
+		videostore.SetLibAVLogLevel("debug")
```

Comment:
Set to `debug` instead of `info` so we can see debug av logs from `concat.c` and `rawsegmenter.c`.